solr/solrj/src/java/org/apache/solr/common/EmptyEntityResolver.java

```diff
@@ -17,15 +17,15 @@
 package org.apache.solr.common;
 
 import java.io.InputStream;
+
+import org.apache.http.impl.io.EmptyInputStream;
```

Comment:
Maybe okay for now but we may soon be looking to not depend needlessly on Apache HttpClient

Reply:
I’m happy to kick this can down the road